superagi/helper/qdrant_helper.py

```diff
@@ -0,0 +1,117 @@
+from sqlalchemy.orm import Session
+from superagi.models.vector_db_config import VectordbConfig
+from qdrant_client import models, QdrantClient
+
+class QdrantHelper:
```

Comment:
We already have these vector dbs in the vector_dbs. These classes does not make sense.